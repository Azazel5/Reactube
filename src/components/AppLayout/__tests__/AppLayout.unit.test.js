import React from 'react'
import {shallow} from 'enzyme'
import AppLayout from '../AppLayout'

describe('<AppLayout />', () => {
    test('renders <AppLayout />', () => {
        const wrapper = shallow(<AppLayout />)
        expect(wrapper).toMatchSnapshot() 
    })

    test('renders <AppLayout /> with one children', () => {
        const wrapper = shallow(
            <AppLayout>
                <div>One child</div>
            </AppLayout>
        )
        expect(wrapper).toMatchSnapshot() 
    })

    test('renders <AppLayout /> with multiple children', () => {
        const wrapper = shallow(
            <AppLayout>
                <div>One child</div>
                <div>
                    <span>Two child</span>
                    <p>Red child, Blue child</p>
                </div>
            </AppLayout>
        )
        expect(wrapper).toMatchSnapshot() 
    })

})