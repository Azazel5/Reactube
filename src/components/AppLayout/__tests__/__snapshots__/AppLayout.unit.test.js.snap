// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLayout /> renders <AppLayout /> 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
</div>
`;

exports[`<AppLayout /> renders <AppLayout /> with multiple children 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    One child
  </div>
  <div>
    <span>
      Two child
    </span>
    <p>
      Red child, Blue child
    </p>
  </div>
</div>
`;

exports[`<AppLayout /> renders <AppLayout /> with one children 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    One child
  </div>
</div>
`;
